a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -188,6 +187,7 @@ def unplug_trunk_subports(self, trunk_id, subports):
             LOG.exception(message)
 
     def get_plugged_parent_port(self, vip):
+        parent_port = None
         try:
             port = self.neutron_client.show_port(vip.port_id)
```

Comment:
(Yes not from this PR, but let's still make the change since we altered function here anyway).

Change to `port = self.get_port(vip.port_id)`

Reply:
We cant do that change here. Actually `self._port_to_parent_port(port.get("port"))` needs a dictionary and we get Port object from `self.get_port(vip.port_id)`. This would need fix in `_port_to_parent_port()` method